test/mitmproxy/test_optmanager.py

```diff
@@ -469,3 +478,119 @@ def test_set():
     opts.process_deferred()
     assert "deferredsequenceoption" not in opts.deferred
     assert opts.deferredsequenceoption == ["a", "b"]
+
+
+def test_relative_script_path(tmpdir):
```

Comment:
Can we please have one test that verifies one `optmanager.load_paths` invocation with a hardcoded config (no need for yaml dumping at runtime - see https://abseil.io/resources/swe-book/html/ch12.html#donapostrophet_put_logic_in_tests), and then a separate `pytest.mark.parametrize`d test for `script_path`? ☺️